baus/utils.py

```diff
@@ -137,21 +137,22 @@ def random_indexes(s, num, replace=False):
 def round_series_match_target(s, target, fillna=np.nan):
     if target == 0 or s.sum() == 0:
         return s
-
-    s = s.fillna(fillna).round().astype('int')
```

Comment:
You changed `s` to `r` here everywhere presumably because leaving at s edits the data that gets passed in or something?